src/bci_build/package/minimal/README.md.j2

```diff
@@ -1,5 +1,11 @@
-# SLE BCI-Minimal: Container image without Zypper
+# SLE BCI-Minimal: Base Container image without Zypper
+{% include 'badges.j2' %}
+SLE BCI-Minimal comes without Zypper, but it does have the RPM package manager installed.
+While RPM can install and remove packages, it lacks support for repositories and automated dependency resolution.
+The SLE BCI-Minimal image is therefore intended for creating deployment containers, and then installing the desired
+RPM packages inside the containers.
 
-SLE BCI-Minimal comes without Zypper, but it does have the RPM package manager installed. While RPM can install and remove packages, it lacks support for repositories and automated dependency resolution. The SLE BCI-Minimal image is therefore intended for creating deployment containers, and then installing the desired RPM packages inside the containers. While you can install the required dependencies, you need to download and resolve them manually. However, this approach is not recommended as it is prone to errors.
+While you can install the required dependencies, you need to download and resolve them manually.
+However, this approach is not recommended as it is prone to errors.
```

Comment:
Hm, I would rather formulate it like this (or something along those lines) to prevent this sounding like "we're shipping an image that will most likely break for you".
```suggestion
However, this approach is not recommended and should only be employed by experienced users.
```

Reply:
this is the official documentation approved text, I just rewrapped it. see https://documentation.suse.com/container/all/single-html/Container-guide/index.html#id-sle-bci-minimal-when-you-do-not-need-zypper

let's do a documentation cleanup in a followup PR, it's difficult enough to get one semantically coherent change together.